apps/sample-app/src/components/Emails.tsx

```diff
@@ -0,0 +1,47 @@
+import { DocumentCard, DocumentCardTitle } from '@fluentui/react';
+import { Text } from '@fluentui/react-components';
+import { Message } from '@microsoft/microsoft-graph-types';
+import React from 'react';
+
+//import { handleOpenMailItem } from './utils';
+//onClick={() => handleOpenMailItem(emailItem)}
```

Comment:
Did you intend for this to be part of your check in?

Reply:
Oops. nope. Thank you for pointing this out to me